frontend/app/assets/css/main.css

```diff
@@ -42,16 +42,24 @@
   }
 }
 
-@utility stroked-text-dark {
+@utility stroked-text-base {
```

Comment:
So intelij sense will suggest `stroked-text-base` but it won't work properly? I think you could just leave this as `stroked-text-1` and override it in `stroked-text-3`